bin/build-docker

```diff
@@ -15,16 +17,16 @@ echo $REPO
 echo $TAG
 echo '--'
 
-# need to pull images for cache_from first
-docker pull $LATEST_TAG || true
-docker pull $COMPOSE_TAG || true
+# Try to pull the latest image for this branch, otherwise pull latest image for caching
+if ! ( docker pull $BRANCH_TAG && docker tag $BRANCH_TAG $COMPOSE_TAG ); then
```

Comment:
Nit: Instead of using COMPOSE_TAG here to specify the image we use for caching, and then using COMPOSE_TAG again later to specify the built docker image, can we use something like CACHE_TAG="$REPO:cached" here? Just don't like re-using names for two different purposes.

Reply:
Yeah this is a good point. I originally re-used the name because I assumed it would auto-cache if we tagged the same name but for some odd reason this didn't work when I tested it. I will make the change.